src/ui/screens/onboarding/ViewOnboarding.qml

```diff
@@ -15,6 +15,14 @@ MZStepNavigation {
     id: stepNav
     objectName: "viewOnboarding"
 
+    //Must be kept in sync with 'views' order (order of progress bar)
```

Comment:
many of the tests already depend on this order. if this order is tampered with, tests will definitely fail. even if you were to reorder items and keep both sections in sync, tests would still fail because they rely on this specific order